tools/travis.ps1

```diff
@@ -1,425 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-param(
-    [ValidateSet('Bootstrap','Build','Failure','Success')]
-    [String]$Stage = 'Build',
-    [String]$NugetKey
-)
-
-Import-Module $PSScriptRoot/../build.psm1 -Force
-Import-Module $PSScriptRoot/packaging -Force
-
-function Send-DailyWebHook
```

Comment:
Do we still need to send the DailyWebHook @TravisEz13?

Reply:
not any more